README.md

```diff
@@ -1 +1,15 @@
-# cloud
+# The ArangoDB Cloud
+
+![ArangoDB Oasis](https://cloud.arangodb.com/assets/logos/arangodb-oasis-logo-whitebg-right.png)
+
+ArangoDB Oasis is the easiest way to run ArangoDB.
```

Comment:
- Available for AWS, Azure and Google Cloud Deployment
- Many regions supported
- Elastic Scalability
- Modern Security
- Special Features to scale with graph, document and search
- [Test it for free](https://cloud.arangodb.com/dashboard)